minio/api.py

```diff
@@ -128,13 +128,15 @@ class Minio(object):
 
     def __init__(self, endpoint, access_key=None,
                  secret_key=None, secure=True,
-                 region=None,
-                 timeout=None,
-                 certificate_bundle=certifi.where()):
+                 region=None, http_client=None):
```

Comment:
Is it possible to do this instead? 

```
default_pool_manager = urllib3.poolmanager.PoolManager()

def __init__ (self, endpoint, access_key=None, 
                     secret_key=None, secure=True,
                     region=None, http_client=default_pool_manager):
```

This would avoid the `if` checks i think.